lib/peep.rb

```diff
@@ -0,0 +1,28 @@
+require 'pg'
+
+class Peep
+  attr_reader :date, :content, :userID
+  
+  def initialize(date, content, userID)
+    @date = date
+    @content = content
+    @userID = userID
+  end
+
+
+  def self.all
+    connection = PG.connect(dbname: 'chitter')
+    result = connection.exec("SELECT * FROM peeps;")
```

Comment:
"SELECT content, date FROM peeps ORDER BY date DESC;"

to order peep first, date after, date descending